app/build.gradle

```diff
@@ -194,33 +194,24 @@ dependencies {
     // Debug with ./gradlew -q app:dependencies --configuration compile
 
     String okHttpVersion = '3.14.0'
-    String supportVersion = '28.0.0'
     String retrofitVersion = '2.5.0'
-    String espressoVersion = '3.0.1'
-    String butterKnifeVersion = '8.8.1'
+    String espressoVersion = '3.1.1'
     String frescoVersion = '1.13.0' // When updating this version, resync file copies under app/src/main/java/com/facebook
-    String testingSupportVersion = '1.0.2'
     String mockitoCore = 'org.mockito:mockito-core:1.9.5'
     String leakCanaryVersion = '1.6.2'
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    implementation "com.android.support:support-compat:$supportVersion"
-    implementation "com.android.support:support-core-utils:$supportVersion"
-    implementation "com.android.support:support-core-ui:$supportVersion"
-    implementation "com.android.support:support-fragment:$supportVersion"
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
-
-    implementation "com.android.support:cardview-v7:$supportVersion"
-    implementation "com.android.support:design:$supportVersion"
-    implementation "com.android.support:recyclerview-v7:$supportVersion"
-    implementation "com.android.support:palette-v7:$supportVersion"
-    implementation "com.android.support:preference-v14:$supportVersion"
-
-    implementation ('com.github.michael-rapp:chrome-like-tab-switcher:0.3.7') {
-        // TODO: remove when this component updates its own dependency on Support 28.0.0
-        exclude group: 'com.android.support'
-    }
+    implementation "com.google.android.material:material:1.0.0"
+    implementation "androidx.core:core:1.0.1"
+    implementation "androidx.cardview:cardview:1.0.0"
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation "androidx.fragment:fragment:1.0.0"
+    implementation "androidx.palette:palette:1.0.0"
+    implementation "androidx.preference:preference:1.0.0"
+    implementation "androidx.recyclerview:recyclerview:1.0.0"
```

Comment:
Lgtm... just one question... should we not have variable for version numbers? is it because they are all different?

Reply:
It actually looks like all of these packages are now decoupled from each other, so certain components might receive updates more often than others.

Reply:
cool! will merge after travis is complete.